sdk/formrecognizer/azure-ai-formrecognizer/tests/test_business_card.py

```diff
@@ -0,0 +1,376 @@
+
+        self.assertEqual(len(business_card.fields.get("Addresses").value), 1)
+        self.assertEqual(business_card.fields.get("Addresses").value[0].value, "2 Kingdom Street Paddington, London, W2 6BD")
+
+        self.assertEqual(len(business_card.fields.get("CompanyNames").value), 1)
+        self.assertEqual(business_card.fields.get("CompanyNames").value[0].value, "Contoso")
+
+    @GlobalFormRecognizerAccountPreparer()
+    @GlobalClientPreparer()
+    def test_business_card_jpg_include_field_elements(self, client):
+        with open(self.business_card_jpg, "rb") as fd:
+            business_card = fd.read()
+        poller = client.begin_recognize_business_cards(business_card, include_field_elements=True)
+
+        result = poller.result()
+        self.assertEqual(len(result), 1)
+        business_card = result[0]
+
+        self.assertFormPagesHasValues(business_card.pages)
+
+        for name, field in business_card.fields.items():
+            if field.value_type not in ["list", "dictionary"]:
+                self.assertFieldElementsHasValues(field.value_data.field_elements, receipt.page_range.first_page_number)
+
+    @GlobalFormRecognizerAccountPreparer()
+    @GlobalClientPreparer()
+    @pytest.mark.live_test_only
+    def test_business_card_continuation_token(self, client):
+
+        with open(self.business_card_jpg, "rb") as fd:
+            business_card = fd.read()
+
+        initial_poller = client.begin_recognize_business_cards(business_card)
+        cont_token = initial_poller.continuation_token()
+        poller = client.begin_recognize_business_cards(business_card, continuation_token=cont_token)
+        result = poller.result()
+        self.assertIsNotNone(result)
+        initial_poller.wait()  # necessary so azure-devtools doesn't throw assertion error
+
+    @GlobalFormRecognizerAccountPreparer()
+    @GlobalClientPreparer(client_kwargs={"api_version": FormRecognizerApiVersion.V2_0})
+    def test_business_card_v2(self, client):
+        with open(self.business_card_jpg, "rb") as fd:
+            business_card = fd.read()
+        with pytest.raises(ValueError) as e:
+            client.begin_recognize_business_cards(business_card)
+        assert "Method 'begin_recognize_business_cards' is only available for API version V2_1_PREVIEW and up" in str(e.value)
+
+    @GlobalFormRecognizerAccountPreparer()
+    @GlobalClientPreparer()
+    def test_business_card_locale_specified(self, client):
+        with open(self.business_card_jpg, "rb") as fd:
+            business_card = fd.read()
+        poller = client.begin_recognize_business_cards(business_card, locale="en-IN")
+        assert 'en-IN' == poller._polling_method._initial_response.http_response.request.query['locale']
+        poller.wait()
+
+    @GlobalFormRecognizerAccountPreparer()
+    @GlobalClientPreparer()
+    def test_business_card_locale_error(self, client):
```

Comment:
Does it makes sense to have BC in multipage? I don't think so... but what do you think

Reply:
I don't think so either, haven't seen an example for one yet